UpgradeGuide.md

```diff
@@ -1,37 +1,30 @@
-
-General releases will list [breaking changes](/BreakingChanges.md) in their descriptions.
-
-## 2017.2 Upgrade Guide for Window Mixed Reality
-
-### What you'll need
-
-- [Unity Editor 2017.2.0p1 MRTP4](http://beta.unity3d.com/download/b1565bfe4a0c/download.html) or later
-- Running the Windows Fall Creator's Update Build 1709 or later
-- Visual Studio 2017 build 15.3 or later
-- Window 10 SDK 10.0.16299.0 or later
-
-### List of breaking changes
-- None to date.  See [Unity's upgrade guide](https://docs.unity3d.com/Manual/UpgradeGuide20172.html) for Unity Specific changes.
-
-## 2017.1 Upgrade Guide
-
-### What you'll need
-
-- [Unity Editor 2017.1.2f1](https://unity3d.com/unity/whats-new/unity-2017.1.2) or later
-- Window Build 1603 or later
-- Visual Studio 2015 or later
-- Windows 10 SDKs 10.0.10240.0 though 10.0.15063.0
-
-### List of breaking changes
-- `TryGetPosition` is now `TryGetPointerPosition`
-- `TryGetOrientation` is now `TryGetPointerRotation`
-- `SpeechKeywordRecognizedEventData` is now `SpeechEventData`
-- All other changes should throw `Obsolete` warnings with suggested updates.
-- Renamed feature folders
-- Merged Test and Example folders
-- Removed `MicStream` scripts and libraries until they pass WACK
-- `GameObjects` that utilize the `WorldAnchorStore` need to have unique names.
-- `SetGlobalListeners` now registers/unregisters during OnEnable/OnDisable.
+# General Release Information
+
+This release is targeted for the Unity 2017.x products 
+- The recommended Unity version is [2017.4](https://unity3d.com/unity/qa/lts-releases?_ga=2.10765437.818138280.1527115303-289721018.1521153098)
+    - The recommended **minimum** Unity version is 2017.1
+        - We will investigate reported issues on Unity 5.6, please file them in [GitHub](https://github.com/Microsoft/MixedRealityToolkit-Unity/issues) and tag them with "Unity 5.6
+- Unity 2017.2 and newer is required for Immersive headset support
+    - Unity 2017.1 can be used for HoloLens projects
+- Windows SDK 10.0.16299 is required for Unity 2017.2 and newer
+- Visual Studio 2017 (15.3 or newer) is required.
+- Windows 10 Fall Creators Update (1709) or newer is required.
+    - Windows 10 April 2018 Update (1803) is recommended 
+
+```
+Note: When upgrading the toolkit in your project, delete the following before importing the new package.
+- Assets\HoloToolkit
+- Assets\HoloToolkit-Examples (if present)
+```
+
+# Testing notes
+
+The Mixed Reality Toolkit team tests on the recommended and the recommended minimum Unity versions.
+- A small amount of testing is performed on other Unity releases
+- HoloToolkit-Examples scenes are used for the majority of testing
+- Compile testing is performed on
+    - .NET Scripting backend
+    - .NET Native (Master configuration in Visual Studio)
```

Comment:
I think we should move the Master configuration to the top and remove Visual Studio all together.